tests/test_phase_psf.py

```diff
@@ -254,67 +282,68 @@ def test_phase_psf_reset():
     # Test frozen AtmosphericScreen first
     atm = galsim.Atmosphere(screen_size=30.0, altitude=10.0, speed=0.1, alpha=1.0, rng=rng)
     aper = galsim.Aperture(diam=1.0, lam=500.0)
-    wf1 = atm.wavefront(aper)
-    atm.advance()
-    wf2 = atm.wavefront(aper)
+    wf1 = atm.wavefront(aper.u, aper.v, None)
+    atm._seek(1.0)
+    wf2 = atm.wavefront(aper.u, aper.v, None)
```

Comment:
I think you say somewhere that public functions should return equal results for two objects that evaluate as equal.  If I've followed everything correctly, then this test seems to violate that condition, right?  `atm` is a nominally immutable object, but here it gives different values depending on an internal state.

I think this means that `wavefront` should not be a public API method.  Is there any reason an end user would want to call this directly rather than use the more user-friendly methods like `makePSF`?